sdk/storage/tests-template.yml

```diff
@@ -49,10 +49,14 @@ stages:
       CloudConfig:
         Preview:
           SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
+          ServiceConnection: azure-sdk-tests
+          SubscriptionConfigurationFilePaths:
+            - eng/common/TestResources/sub-config/AzurePublicMsft.json
```

Comment:
If this is for Preview sub you will need to do something like https://github.com/Azure/azure-sdk-for-net/pull/44427/files#diff-3321836f818e9ee74bb58c3b303946aab1054783000f8e492e35445ed540dcc4R16 to get the correct configuration. Right now I'm not even sure that it is necessary at all so you might even consider removing this for now as the subscription configuration should still be coming in from `$(sub-config-azure-cloud-test-resources-preview)`